README.md

```diff
@@ -0,0 +1,30 @@
+Docker Images Generation
+========================
+
+The `build-img` script can generate images for a particular distro based on only one Dockerfile.
+
+The `docker/` directory contains all the script needed to use when building images.
+
+Please take a look to the Makefile for configuration options.
+
+make, make all and make build will build the docker images locally. If no TAG variable is specified, then only the images with the form ORG/IMG:DIST-vX will be generated, as well as the ORG/IMG which will only be tagged for the LATEST\_DIST distribution.
+
+If make build TAG=v2.0.1 is used (for example), the image ORG/IMG:DIST-v2.0.1 will be tagged too.
+
+If a TAG is specified but is a pre-release (as per SemVer, i.e. it contains a -), the default tagged image will be ORG/IMG:DIST-vX-test (instead of ORG/IMG:DIST-vX) and ORG/IMG will not be tagged at all.
+
+make push will push all the images that would be generated by make build (same rules apply for TAG).
+
+Docker Images Publishing
+========================
+
+Images are automatically published for tags via [Travis](https://travis-ci.org/sociomantic-tsunami/cachalot) to [DockerHub](https://travis-ci.org/sociomantic-tsunami/cachalot/settings)'s [sociomantictsunami](https://cloud.docker.com/u/sociomantictsunami/repository/list) organization.
+
```

Comment:
I've got 404  (page not found) on the last link), is the correct link?

Reply:
Yes, you don't have permissions to access the settings page, we probably have to change that.